jazzygrid/theme.json

```
@@ -0,0 +1,713 @@
+					},
+					"outline": {
+						"color": "var(--wp--preset--color--primary)",
+						"offset": "2px",
+						"style": "dotted",
+						"width": "1px"
+					}
+				},
+				":hover": {
+					"color": {
+						"background": "var(--wp--preset--color--secondary)",
+						"text": "var(--wp--preset--color--background)"
+					}
+				},
+				"border": {
+					"radius": "0.25rem"
+				},
+				"color": {
+					"background": "var(--wp--preset--color--primary)",
+					"text": "var(--wp--preset--color--foreground)"
+				}
+			},
+			"caption": {
+				"color": {
+					"text": "var(--wp--preset--color--foreground)"
+				}
+			},
+			"h1": {
+				"typography": {
+					"fontFamily": "var(--wp--preset--font-family--d-din-condensed)",
+					"fontSize": "var(--wp--preset--font-size--x-large)",
+					"lineHeight": "1"
+				}
+			},
+			"h2": {
+				"typography": {
+					"fontFamily": "var(--wp--preset--font-family--d-din-condensed)",
+					"fontSize": "var(--wp--preset--font-size--large)",
+					"lineHeight": "1"
+				}
+			},
+			"h3": {
+				"typography": {
+					"fontFamily": "var(--wp--preset--font-family--d-din-condensed)",
+					"fontSize": "clamp(1.5rem, calc(1.5rem + ((1vw - 0.48rem) * 0.4808)), 1.75rem)",
+					"lineHeight": "1"
+				}
+			},
+			"h4": {
+				"typography": {
+					"fontFamily": "var(--wp--preset--font-family--d-din-condensed)",
+					"fontSize": "var(--wp--preset--font-size--medium)",
+					"lineHeight": "1"
+				}
+			},
+			"h5": {
+				"typography": {
+					"fontFamily": "var(--wp--preset--font-family--d-din-condensed)",
+					"fontSize": "var(--wp--preset--font-size--medium)",
+					"lineHeight": "1"
+				}
+			},
+			"h6": {
+				"typography": {
+					"fontFamily": "var(--wp--preset--font-family--d-din-condensed)",
+					"fontSize": "var(--wp--preset--font-size--small)",
+					"lineHeight": "1"
+				}
+			},
+			"heading": {
+				"color": {
+					"text": "var(--wp--preset--color--foreground)"
+				},
+				"typography": {
+					"fontFamily": "var(--wp--preset--font-family--d-din-condensed)",
+					"fontStyle": "normal",
+					"fontWeight": "400",
+					"letterSpacing": "rem",
+					"lineHeight": "1.125",
+					"textTransform": "uppercase"
+				}
+			},
+			"link": {
+				":hover": {
+					"typography": {
+						"textDecoration": "none"
+					}
+				},
+				"typography": {
+					"fontStyle": "normal",
+					"fontWeight": "700",
+					"textDecoration": "none"
+				}
+			}
+		},
+		"spacing": {
+			"blockGap": "1rem",
+			"padding": {
+				"bottom": "0px",
+				"left": "0px",
+				"right": "0px",
+				"top": "0px"
+			}
+		},
+		"typography": {
+			"fontFamily": "var(--wp--preset--font-family--rubik)",
+			"fontSize": "var(--wp--preset--font-size--medium)",
+			"letterSpacing": "0rem",
+			"lineHeight": "1.4"
+		}
+	},
+	"templateParts": [
+		{
+			"area": "header",
+			"name": "header"
+		},
+		{
+			"area": "footer",
+			"name": "footer"
+		}
+	],
+	"version": 3,
+	"$schema": "https://schemas.wp.org/trunk/theme.json"
```

Should match the minimum WP version to make sure all features are actually available. Same goes for style variations.

```suggestion
	"$schema": "https://schemas.wp.org/wp/6.0/theme.json"
```

There are a lot of things here that aren't available to WP 6.0 that need to get changed, or the minimum version needs to be updated.